### ISSUE TITLE
Problem: Adding S3 storage support requires editing config files

### ISSUE THREAD

**Issue by jorgesat** [magicstone-dev/ecko#483]
Covers, avatars and other media attachments often consume a lot of space. It would be more convenient to store them as S3 objects using third party providers. However, configuring S3 requires editing the `.env.production` file. To improve this configuration process, I suggest to add a section to the Administration settings where the S3 storage support can be configured by admins.

**Comment by weex** [comment]
I agree that such a change should be possible without editing configs. Is there a data migration process that needed to go along with the config change?
